jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/Tabs.java

```diff
@@ -0,0 +1,100 @@
+package com.epam.jdi.light.angular.elements.complex;
+
+import com.epam.jdi.light.angular.asserts.TabsAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.interfaces.common.IsText;
+
+import java.util.List;
+
+public class Tabs extends UIBaseElement<TabsAssert> implements IsText {
+
+    /*TABS */
+    private WebList getTabs() {
+        return this.finds("[role='tab']");
+    }
+
+    private UIElement getTabByNumber(int tabNumber) {
```

Comment:
According to the convention's recommendations, private methods should be located at the end of the class.